### PR TITLE
Implement Sentry on the API 

### PR DESCRIPTION
**Pull request by cdvx**
Resolves:https://github.com/opencollective/opencollective/issues/2997

chore(api-errors): Implement Sentry on the API
 - Implement sentry for rich error reporting
[resolves 2997]

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "npm": "6.12.0"
   },
   "dependencies": {
-    "@babel/core": "7.9.0",
```
Comment (znarf) [inline package.json]: Please double check all changes to package.json and make sure you're only adding sentry and not changing anything else.

Comment (cdvx) [reply package.json]: Got it 👍🏾

Comment (cdvx) [reply package.json]: @znarf ,this has been resolved

**Comment by Betree** [comment]
Also @cdvx it would be great if you could rebase on master

**Comment by cdvx** [comment]
> Also @cdvx it would be great if you could rebase on master

Great, thanks for the feedback @Betree , let me work on this and ping you

**Comment by cdvx** [comment]
> Also @cdvx it would be great if you could rebase on master

@Betree , the PR has been rebased with master and sentry error reporting has also been assed to `graphqlServerV2`

**Comment by Betree** [comment]
Let's merge & try on staging. Thanks @cdvx!

**Comment by Betree** [comment]
@cdvx  You've been added to https://github.com/orgs/opencollective/teams/contributors/members. **Congrats!** :tada: You can now push directly to most Open Collective repos (only the `master` branches are protected).

**Comment by cdvx** [comment]
> @cdvx You've been added to https://github.com/orgs/opencollective/teams/contributors/members. **Congrats!** 🎉 You can now push directly to most Open Collective repos (only the `master` branches are protected).

Great, Thank you so much @Betree, really glad to be a contributor! 💻

**Pull request merged**